api/src/pcapi/routes/backoffice/templates/admin/roles.html

```diff
@@ -1,32 +1,50 @@
 {% block main_content %}
   <div class="container">
     <h2>Gestion des rôles</h2>
-    {% for form in forms %}
-      <div class="card mb-4">
-        <div class="card-header"
-             data-bs-toggle="collapse"
-             href=".pc-collapse-{{ form.name }}"
-             role="button">{{ form.name }}</div>
-        <div class="collapse card-collapse p-3 shadow pc-collapse-{{ form.name }}">
-          <form class=".{{ form.name }}"
-                action="{{ url_for(".update_role", role_id=form.id) }}"
-                method="post"
-                name="{{ url_for(".update_role", role_id=form.id) | action_to_name }}">
-            <div class="card-body">
-              <div class="row">
-                {% for form_field in forms[form] %}{{ form_field }}{% endfor %}
-              </div>
-              <div class="row">
-                <div class="my-3">
-                  <button type="submit"
-                          class="btn btn-sm py-2 btn-primary">Sauvegarder les modifications</button>
-                </div>
+    <div class="mt-4">
+      {% call tabs.build_details_tabs_wrapper() %}
+        {{ tabs.build_details_tab("gestion", "Gestion", True) }}
+        {{ tabs.build_details_tab("historique", "Historique") }}
+      {% endcall %}
+      {% call tabs.build_details_tabs_content_wrapper() %}
+        {% call tabs.build_details_tab_content("gestion", True) %}
+          {% for form in forms %}
+            <div class="card mb-4">
+              <div class="card-header"
+                   data-bs-toggle="collapse"
+                   href=".pc-collapse-{{ form.name }}"
+                   role="button">{{ form.name }}</div>
+              <div class="collapse card-collapse p-3 shadow pc-collapse-{{ form.name }}">
+                <form class=".{{ form.name }}"
+                      action="{{ url_for(".update_role", role_id=form.id) }}"
+                      method="post"
+                      name="{{ url_for(".update_role", role_id=form.id) | action_to_name }}">
+                  <div class="card-body">
+                    <div class="row">
+                      {% for form_field in forms[form] %}{{ form_field }}{% endfor %}
+                    </div>
+                    <div class="row">
+                      <div class="my-3">
+                        <button type="submit"
+                                class="btn btn-sm py-2 btn-primary">Sauvegarder les modifications</button>
+                      </div>
+                    </div>
+                  </div>
+                </form>
               </div>
             </div>
-          </form>
-        </div>
-      </div>
-    {% endfor %}
-  </div>
-{% endblock main_content %}
+          {% endfor %}
+        {% endcall %}
+        {% call tabs.build_details_tab_content("historique") %}
+          <div class="mt-4">
+            <turbo-frame data-turbo="false" id="admin_role_details_frame" src="{{ url_for("backoffice_web.get_roles_history") }}">
```

Comment:
Essayons de garder, comme ailleurs, l'`id` cohérent avec le contenu : 
```suggestion
            <turbo-frame data-turbo="false" id="admin_roles_history_frame" src="{{ url_for("backoffice_web.get_roles_history") }}">
```
À modifier aussi dans roles_history.html.